designer/server/src/main/scala/pl/touk/nussknacker/ui/process/repository/DBIOActionRunner.scala

```diff
@@ -18,6 +20,23 @@ class DBIOActionRunner(dbRef: DbRef) {
 
   def run[T](action: DB[T]): Future[T] =
     dbRef.db.run(action)
+
+  def safeRunInTransaction[T, E](action: DB[T], error: Throwable => E)(
+      implicit ec: ExecutionContext
+  ): Future[Either[E, T]] =
+    safeRun(action.transactionally, error)
+
+  def safeRun[T, E](action: DB[T], error: Throwable => E)(
+      implicit ec: ExecutionContext
+  ): Future[Either[E, T]] =
+    run(action).transformWith {
+      case Failure(ex) => {
+        logger.warn("Exception occurred during database access", ex)
```

Comment:
I don't think we need this error. We can handle the exception at the business level somehow and the exception in the log will be misleading

Reply:
Done